### PR TITLE
Reorganizing demo app #no-public-changes

### PR DESCRIPTION
**Pull request by tbaranes**
Related to #484 and #345 

- Grouping files by features 
- Finder directories maps xcodeproj
- A files renaming: 
   - UserInterfaceActivityIndicatorViewController -> ActivityIndicatorViewController
   - PresentationPresentedViewController -> PresentedViewController
- A few typo fixed (missing closed parenthesis in transition names, ...)
- Add missing final
- Removing public from demo app (the few I see, there's probably more)

### CODE REVIEW

File: IBAnimatableApp/IBAnimatableApp/Playground/Transitions/TransitionTableViewController.swift [hunk 1]
```diff
@@ -185,9 +185,9 @@ fileprivate extension TransitionAnimationType {
   var asString: String {
     switch self {
     case .fade(let direction):
-      return "fade" + "(\(direction.rawValue)"
+      return "fade" + "(\(direction.rawValue))"
```
Comment (JakeLin) [inline IBAnimatableApp/IBAnimatableApp/Playground/Transitions/TransitionTableViewController.swift:188]: It was a bug 😱

File: IBAnimatableApp/IBAnimatableApp/Playground/ActivityIndicators/ActivityIndicatorViewController.swift [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import UIKit
 import IBAnimatable
 
-class UserInterfaceActivityIndicatorViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource {
+final class ActivityIndicatorViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource {
```
Comment (JakeLin) [inline IBAnimatableApp/IBAnimatableApp/Playground/ActivityIndicators/ActivityIndicatorViewController.swift:9]: Good one.

**Review by JakeLin — APPROVED** [review]
👍 Cool, better to newcomers to pick it up. Thanks.

**Comment by tbaranes** [comment]
There's still rooms for improvements around this, but it's a good first step!

File: IBAnimatableApp/IBAnimatableApp/Playground/Transitions/TransitionTableViewController.swift (hunk 1 above)
Comment (SD10) [reply IBAnimatableApp/IBAnimatableApp/Playground/Transitions/TransitionTableViewController.swift:188]: My bug too 😱

**Pull request merged**